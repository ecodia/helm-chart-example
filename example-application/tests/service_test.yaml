suite: test service
templates:
  - service.yaml
tests:
  - it: should be of type Service
    asserts:
      - isKind:
          of: Service
  - it: manifest should match snapshot
    set:
      image:
        repository: test-docker-registry
        tag: 1.2.3-TEST
    asserts:
      - matchSnapshot:
          path: spec
