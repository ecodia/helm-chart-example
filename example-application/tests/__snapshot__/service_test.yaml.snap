manifest should match snapshot:
  1: |
    ports:
    - name: http
      port: 80
      protocol: TCP
      targetPort: http
    selector:
      app.kubernetes.io/instance: RELEASE-NAME
      app.kubernetes.io/name: example-application
    type: ClusterIP
